gatling-jms/src/main/scala/io/gatling/jms/request/JmsRequestBuilder.scala

```diff
@@ -24,29 +24,30 @@ import io.gatling.jms._
 import io.gatling.jms.action.JmsReqReplyBuilder
 
 import com.softwaremill.quicklens._
+import io.gatling.jms.action.JmsRequestSendBuilder
 
 case class JmsRequestBuilderBase(requestName: String) {
 
   def reqreply(implicit configuration: GatlingConfiguration) = JmsRequestBuilderQueue(requestName, JmsReqReplyBuilder.apply(_, configuration))
+
+  def send(implicit configuration: GatlingConfiguration) = JmsRequestBuilderQueue(requestName, JmsRequestSendBuilder.apply(_, configuration))
```

Comment:
I will change these Builder to JmsRequestReplyBuilderQueue and JmsRequestSendBuilderQueue. So I can apadt the exposed methods.